packages/fuels-contract/tests/harness.rs

```diff
@@ -0,0 +1,44 @@
+use fuels::prelude::{launch_provider_and_get_single_wallet, Contract, TxParameters};
```

Comment:
You should put these tests in `contract.rs`. These are "unit tests" whereas we consider the "harness" naming to be more E2E tests.